playbooks/roles/ses-osh/defaults/main.yml

```diff
@@ -1,2 +1,18 @@
 ---
 radosgw_keystone: False
+
+# Openstack-helm services expects a specific application name which is
+# different from what's created by ses openstack.integrate script.
+# So as interim solution, creating pools with different names to allow
+# exected application names to be enabled with those pools.
+
+socok8s_openstack_glance_pool_name: socok8s_images
```

Comment:
doesn't have to be in this commit, but it'd be nice to have pool names tied to the workspace env variable. if deploy socok8s to a different workspace, I think would be a good idea that the ses pools are also separated.

Reply:
Not sure about the need of having pool name tied to workspace name. 
Purpose for this change is to have different pool name from integrate script which having socok8s_ prefix provides that. Actually having different but fixed name provides predictability in documentation and usage/debugging perspective like we have with ucp and openstack namespaces even though they can be different if needed.

Reply:
Added support for prefix but for now, keeping the same names as used previously to have minimal changes for TP release deadline.